backends/v3/src/backend.rs

```diff
@@ -112,6 +136,10 @@ impl Backend for BackendV3 {
         .is_ok()
     }
 
+    fn events(&self) -> BroadcastReceiver<EngineState> {
+        self.state_events.0.subscribe()
```

Comment:
You should send the current state as soon as a new connection is open (new subscribe)